0xturner/gpt-applicant-assistant
# Parse and validate the request + setup vitest

Validation steps:
1. Parse type with Zod
2. Check token count
3. Send to OpenAI moderation endpoint
